tools/bin/mtest

```diff
@@ -397,36 +432,57 @@ def configure_options(target):
     # user-specified options to apply to all targets
     options += args.config_options
 
+    set_no_platform_entropy = False
     disable_spm = False
 
     individual_options = [x for x in options if x.startswith("MBEDTLS_") or x.startswith("-MBEDTLS_") or x.startswith("PSA_") or x.startswith("-PSA_")]
     bulk_options = [x for x in options if x not in individual_options]
 
     for config in bulk_options:
-        if config.endswith(".h"):
-            # a config file - overwrite existing config file
-            path = config if os.path.exists(config) else f"{ROOT}/configs/{config}"
-            if path.split("/")[-1].startswith("crypto_config_"):
-                dest = CRYPTO_CONFIG_FILE
+        if config == COPY_TFM_CONFIG:
+            # special flag to copy both TF-M config files - these have a couple of possible locations
+            for mc, cc in TFM_CONFIG_FILES:
+                if os.path.exists(f"{ROOT}/configs/{mc}") and (cc is None or os.path.exists(f"{ROOT}/configs/{cc}")):
+                    if mc == TFM_CONFIG_FILES[0][0]:
+                        # latest TF-M config file unsets this, so set it again
+                        set_no_platform_entropy = True
```

Comment:
note: #8164 moves the (verbatim) TFM config files to a directory, and has a file `config-tfm.h` that will apply the things like this

Reply:
The wrapper disables `MBEDTLS_NO_PLATFORM_ENTROPY`, which we want to set for TF-M size measurement.